src/AreaChart/AreaSeries/AreaSeries.tsx

```diff
@@ -272,17 +280,34 @@ export const AreaSeries: FC<Partial<AreaSeriesProps>> = ({
     );
   }, [activePoint, activeValues, height, markLine]);
 
+  const renderMarker = useCallback(() => {
+    return (
+      <Fragment>
+        {marker && (
+          <CloneElement<MarkerProps>
+            element={marker}
+            width={width}
+            pointY={marker.props.pointY}
```

Comment:
The clone element will project the `marker.props` automatically so u shouldn't need to pass them manually.